vc.adoc

```diff
@@ -1076,6 +1087,152 @@ Content-Type: application/json;charset=UTF-8
+----
+
+[[_example_response_group_exp]]
+==== link:#_example_response_group_exp[Example response]
+
+[source,highlightjs,highlight,nowrap]
+----
+HTTP/1.1 200 OK
+Content-Length: 417
+Content-Type: application/json;charset=UTF-8
+
+{
+   "items":[
+      {
+         "groupexp_id":"9204039012020",
+         "images":[
+            {
+               "format":"large",
+               "sourceUrl":"http://master.y360cms.demo.fivejars.com/sites/default/files/styles/large/public/downtown_exterior_0.jpg?itok=yaosJs2t"
+            },
+            {
+               "format":"medium",
+               "sourceUrl":"http://master.y360cms.demo.fivejars.com/sites/default/files/styles/medium/public/downtown_exterior_0.jpg?itok=yaosJs2t"
+            },
+            {
+               "format":"small",
+               "sourceUrl":"http://master.y360cms.demo.fivejars.com/sites/default/files/styles/small/public/downtown_exterior_0.jpg?itok=yaosJs2t"
+            }
+         ],
+         "video":{
```

Comment:
Let's incorporate a response example, where optional attributes are not presented

Reply:
fixed